modules/topicsFpdModule.js

```diff
@@ -61,7 +93,9 @@ export function getTopics(doc = document) {
 const topicsData = getTopics().then((topics) => getTopicsData(getRefererInfo().domain, topics));
 
 export function processFpd(config, {global}, {data = topicsData} = {}) {
+  loadTopicsForBidders();
```

Comment:
this runs on each auction, and will attach a new message listener each time.

Reply:
Added Check @dgirardi .